contracts/Migrations.sol

```diff
@@ -10,15 +10,15 @@ contract Migrations {
             _;
     }
 
-    function Migrations() {
+    function Migrations() public {
```

Comment:
These methods that are marked `public`...they're `public` by default and we're just adding this to be explicit, right? Good idea.

Reply:
Yep, the compiler also emits warnings if you don't add explicit visibility declarations - a non-functionality related todo is to do clean up any compiler warnings and make the CI fail if we have any warnings